.github/workflows/continuous-integration.yml

```
@@ -153,6 +153,87 @@ jobs:
+        id: cache-bootstrap
+        uses: actions/cache/restore@v4
+        with:
+          path: |
+            ./ledger.preprod.db
+            ./chain.preprod.db
+          key: bootstrap-state-${{ runner.os }}-preprod-${{ env.LEDGER_CACHE_VERSION }}
+
+      - name: Start Amaru
+        id: amaru
+        if: steps.cache-bootstrap.outputs.cache-hit == 'true'
+        background: true
```

actionlint (1.7.12)

[error] unexpected key "background" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)